### ISSUE TITLE
Add a way to set pass options

### ISSUE THREAD

**Issue by phated** [grain-lang/binaryen.ml#191]
Binaryen has some optimization options, such as `closedWorld`, which affect some optimization passes. We should add an API to set these options.

**Comment by peblair** [comment]
@phated I don't believe this is (currently) possible in the Binaryen C API. Did you have an idea for how to tackle this?
